Sources/Helpers/OAUtilities.m

```diff
@@ -2179,4 +2180,45 @@ + (NSString *) getFormattedValue:(NSString *)value unit:(NSString *)unit separat
     return [NSString stringWithFormat:separateWithSpace ? OALocalizedString(@"ltr_or_rtl_combine_via_space") : @"%@%@", value, unit];
 }
 
++ (NSString *) buildGeoUrl:(NSString *)latitude longitude:(NSString *)longitude zoom:(int)zoom
```

Comment:
It is way better to pass doubles here and format them to the fifth floating number for consistency.